### ISSUE TITLE
Called getBlockState() without a targeted block

### ISSUE THREAD

**Issue by maxvar** [Crendgrim/DynamicCrosshairCompat#14]
With the latest versions of main/compat sill errs sometimes.
Stacktrace (repeated13 times within a few seconds, then trice again in ~30 minutes):
```
[20:37:55] [Render thread/ERROR]: Encountered invalid context state: 
mod.crend.dynamiccrosshair.api.InvalidContextState: Called getBlockState() without a targeted block!
	at mod.crend.dynamiccrosshair.api.CrosshairContext.getBlockState(CrosshairContext.java:117) ~[dynamiccrosshair-5.0+1.19.3-fabric.jar:?]
	at mod.crend.dynamiccrosshair.api.CrosshairContext.getBlock(CrosshairContext.java:126) ~[dynamiccrosshair-5.0+1.19.3-fabric.jar:?]
	at mod.crend.dynamiccrosshair.compat.techreborn.RebornCoreHandler.checkUsableItem(RebornCoreHandler.java:54) ~[dynamiccrosshair-compat-3.1+1.19.3.jar:?]
	at mod.crend.dynamiccrosshair.compat.techreborn.ApiImplTechReborn.computeFromItem(ApiImplTechReborn.java:82) ~[dynamiccrosshair-compat-3.1+1.19.3.jar:?]
	at mod.crend.dynamiccrosshair.component.CrosshairHandler.buildCrosshairAdvancedFromItem(CrosshairHandler.java:60) ~[dynamiccrosshair-5.0+1.19.3-fabric.jar:?]
	at mod.crend.dynamiccrosshair.component.CrosshairHandler.buildCrosshairAdvancedByHand(CrosshairHandler.java:88) ~[dynamiccrosshair-5.0+1.19.3-fabric.jar:?]
	at mod.crend.dynamiccrosshair.component.CrosshairHandler.buildCrosshairAdvanced(CrosshairHandler.java:93) ~[dynamiccrosshair-5.0+1.19.3-fabric.jar:?]
	at mod.crend.dynamiccrosshair.component.CrosshairHandler.buildCrosshairDynamic(CrosshairHandler.java:168) ~[dynamiccrosshair-5.0+1.19.3-fabric.jar:?]
	at mod.crend.dynamiccrosshair.component.CrosshairHandler.buildCrosshair(CrosshairHandler.java:222) ~[dynamiccrosshair-5.0+1.19.3-fabric.jar:?]
	at mod.crend.dynamiccrosshair.component.CrosshairHandler.checkShowCrosshair(CrosshairHandler.java:262) ~[dynamiccrosshair-5.0+1.19.3-fabric.jar:?]
	at mod.crend.dynamiccrosshair.component.CrosshairHandler.tick(CrosshairHandler.java:283) ~[dynamiccrosshair-5.0+1.19.3-fabric.jar:?]
	at net.minecraft.class_329.handler$bbg000$dynamiccrosshair$tickDynamicCrosshair(class_329.java:7048) ~[client-intermediary.jar:?]
	at net.minecraft.class_329.method_1748(class_329.java:1311) ~[client-intermediary.jar:?]
	at net.minecraft.class_329.method_39191(class_329.java:1275) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1574(class_310.java:1826) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1150) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:781) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.method_44604(Main.java:244) ~[minecraft-1.19.3-client.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:51) ~[minecraft-1.19.3-client.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461) ~[fabric-loader-0.14.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.14.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.14.12.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:210) ~[NewLaunch.jar:?]
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:245) ~[NewLaunch.jar:?]
	at org.multimc.EntryPoint.listen(EntryPoint.java:143) ~[NewLaunch.jar:?]
	at org.multimc.EntryPoint.main(EntryPoint.java:34) ~[NewLaunch.jar:?]
```
what's involved:
```
Mods:
  [✔️] advancementinfo-1.19.3-fabric0.68.1-1.3.1
  [✔️] advdebug-2.3.0
  [✔️] appleskin-fabric-mc1.19.3-2.4.2
  [✔️] appliedenergistics2-fabric-13.0.6-beta
  [✔️] architectury-7.0.66-fabric
  [✔️] AttributeFix-Fabric-1.19.3-18.0.1
  [✔️] AutoAttack-1.2.3_Fabric-1.19.3
  [✔️] autoswitch-6.0.0
  [✔️] badpackets-fabric-0.3.4
  [✔️] balm-fabric-1.19.3-5.0.1
  [✔️] better-infinity-1.0.0
  [✔️] blockus-2.6.1+1.19.3
  [✔️] Bookshelf-Fabric-1.19.3-17.0.3
  [✔️] borderless-mining-1.1.6+1.19.3
  [✔️] cardinal-components-api-5.1.0
  [✔️] CatEyes-5.2.3_Fabric-1.19.3
  [✔️] Chunky-1.3.53
  [✔️] cloth-config-9.0.94-fabric
  [✔️] Clumps-fabric-1.19.3-9.0.0+16
  [✔️] collective-1.19.3-6.22
  [✔️] continuity-3.0.0-alpha.21.19.3
  [✔️] Controlling-fabric-1.19.3-10.0+9
  [✔️] Couplings-1.9.2+1.19.3
  [✔️] dark-enchanting-1.0.1+1.19.3
  [✔️] diggusmaximus-1.5.8-1.19.3
  [✔️] durabilityviewer-1.19.3-fabric0.68.1-1.10.3
  [✔️] dynamiccrosshair-5.0+1.19.3-fabric
  [✔️] dynamiccrosshair-compat-3.1+1.19.3
  [✔️] easiervillagertrading-1.19.3-fabric0.68.1-1.5.4
  [✔️] EnchantmentDescriptions-Fabric-1.19.3-14.0.6
  [✔️] expandedstorage-9.0.0-alpha.2+1.19.3-fabric
  [✔️] fabric-api-0.72.0+1.19.3
  [✔️] fabric-language-kotlin-1.9.0+kotlin.1.8.0
  [✔️] fabric-language-scala-1.1.0+scala.2.13.6-fat
  [✔️] fallingleaves-1.14.4+1.19.3
  [✔️] FallingTree-1.19.3-3.11.1
  [✔️] grid-1.19.3-fabric0.68.1-1.6.7
  [✔️] IBEEditor-1.19.3-2.2.2-fabric
  [✔️] idwtialsimmoedm-0.2.0+1.19.3
  [✔️] immersive-portals-2.4.8-mc1.19.3-fabric
  [✔️] indium-1.0.12+mc1.19.3-LOCAL
  [✔️] infinitevillagertrading-1.19.3-fabric0.68.1-1.1
  [✔️] inmis-2.8.0-1.19.3
  [✔️] invhud.fabric.1.19-3.4.7
  [✔️] iris-mc1.19.3-1.5.1
  [✔️] jumpoverfences-1.16-0.0.3+icon
  [✔️] Kiwi-1.19.3-fabric-9.1.1
  [✔️] LargeFluidTank-19.2.0-fabric
  [✔️] LightingWand-1.19.3-fabric-6.0.0
  [✔️] lithium-fabric-mc1.19.3-0.10.4
  [✔️] mcf-easyfeed-1.4.0-1.19.3
  [✔️] mcf-magnets-1.3.0-1.19.3
  [✔️] mcf-reap-1.7.1-1.19.3
  [✔️] minekraftmod-1.4.0-1.19.3
  [✔️] modmenu-5.0.2
  [✔️] mousewheelie-1.10.7+mc1.19.3
  [✔️] noenchantcap-fabric-1.19.3-2.2.6
  [✔️] observable-4.3.1
  [✔️] passiveshield-1.19.3-3.0
  [✔️] PerfectAccuracy-1.2-1.19
  [✔️] PresenceFootsteps-1.7.1
  [✔️] quartz-elevator-2.2.1+1.19
  [✔️] raised-fabric-1.19.3-1.1.4
  [✔️] RebornCore-5.5.0
  [✔️] RoughlyEnoughItems-10.0.581
  [✔️] ScalableCatsForce-2.13.10-build-15-with-library
  [✔️] sodium-fabric-mc1.19.3-0.4.8+build.22
  [✔️] TechReborn-5.5.0
  [✔️] toms_storage_fabric-1.19.3-1.4.6
  [✔️] toolleveling-fabric-1.19.3-1.4.3
  [✔️] travelers-backpack-1.19.3-8.3.0
  [✔️] trinkets-3.5.1
  [✔️] worldedit-mod-7.2.13
  [✔️] wthit-fabric-6.1.4
  [✔️] Xaeros_Minimap_22.17.1_Fabric_1.19.3
  [✔️] XaerosWorldMap_1.28.8_Fabric_1.19.3
```

**Comment by Crendgrim** [comment]
Thank you for the report! Will be fixed in the next version. Does it impede gameplay in any way or just throw out the errors?

**Comment by maxvar** [comment]
> Thank you for the report! Will be fixed in the next version. Does it impede gameplay in any way or just throw out the errors?

it only spams the logs, no other issues

**Comment by Crendgrim** [comment]
Finally fixed with versions 1.19+3.2 and 1.19.3+3.1.1
